spec.html

```diff
@@ -39344,18 +39344,15 @@ <h1>
             1. If _targetOffset_ is +&infin;, throw a *RangeError* exception.
             1. If _srcLength_ + _targetOffset_ &gt; _targetLength_, throw a *RangeError* exception.
             1. If _target_.[[ContentType]] &ne; _source_.[[ContentType]], throw a *TypeError* exception.
-            1. If both IsSharedArrayBuffer(_srcBuffer_) and IsSharedArrayBuffer(_targetBuffer_) are *true*, then
-              1. If _srcBuffer_.[[ArrayBufferData]] and _targetBuffer_.[[ArrayBufferData]] are the same Shared Data Block values, let _same_ be *true*; else let _same_ be *false*.
-            1. Else, let _same_ be SameValue(_srcBuffer_, _targetBuffer_).
-            1. If _same_ is *true*, then
+            1. If _srcBuffer_ is _targetBuffer_ or IsSharedArrayBuffer(_srcBuffer_) is *true*, IsSharedArrayBuffer(_targetBuffer_) is *true*, and _srcBuffer_.[[ArrayBufferData]] is _targetBuffer_.[[ArrayBufferData]], then
```

Comment:
For this to have the same meaning as before, the reader must parse "if x or y, z, and w" as "if (x) or (y, z, and w)" rather than "if (x or y), z, and w". I definitely do not want to rely on the reader doing that parse.

(I didn't parse it that way at first and was in the process of writing a comment about this not being equivalent before I figured out what you'd intended.)

Also it's too much stuff to go on one line. The previous thing was fine.